docker-compose.yml

```diff
@@ -19,20 +18,12 @@ services:
 
   php:
```

Comment:
Can we rename this to "app" or something? Since it's no longer *just* the PHP container, but also the webserver.